README.md

```diff
@@ -26,269 +24,224 @@ pip install jupyterhub-ltiauthenticator
 
-2.  [Enable LTI Components](http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/lti_component.html#enabling-lti-components-for-a-course)
-    for your course.
+Start by following the steps below to configure your JupyterHub setup with the basic settings. Then, navigate to your LMS vendor's section to complete the installation and configuration steps.
 
-3.  Create a _client key_ and _client secret_ for use by edX to authenticate your hub. Open up any command line:
+> **Note**: if you LMS is not listed feel free to send us a PR with instructions for this new LMS.
 
-    1.  Run `openssl rand -hex 32` and save the output. This will be your LTI Client **Key**.
-    2.  Run `openssl rand -hex 32` and save the output. This will be your LTI Client **Secret**.
+There are two configuration values to define with LTI 1.1:
 
-    Now we will use these strings to allow edX and JupyterHub to authenticate each other.
+- The **consumers** setting: the value for the LTI 1.1 configuration `consumer key` and `shared secret`.
+- The **username_key** setting: the parameter from the LTI 1.1 launch request used to set the JupyterHub username.
+- The **config_description** setting:
+- The **config_icon** setting:
+- The **config_title** setting:
 
-    _Note_: These commands will simply generate strings for you, it will not store them anywhere on the computer. Therefore you do not need to run these commands on your JupyterHub server--we will be supplying them manually in the next few steps.
+| LTI Authenticator Configuration Setting | Required | Description                                                              | Default                            |
+| --------------------------------------- | -------- | ------------------------------------------------------------------------ | ---------------------------------- |
+| config_description                      | No       | The LTI 1.1 external tool description                                    | `JupyterHub LTI 1.1 external tool` |
+| config_icon                             | No       | The http/s URL with the LTI 1.1 icon                                     | `nil`                              |
+| config_title                            | No       | The LTI 1.1 external tool Title                                          | `JupyterHub`                       |
+| consumers                               | Yes      | The key/value pair that represents the client key and shared secret      | `{}`                               |
+| username_key                            | No       | The LTI 1.1 launch parameter that contains the JupyterHub username value | `canvas_custom_user_id`            |
```

Comment:
This section is confusing to me.

1. "There are two configuration values to define with LTI 1.1:"
   I wonder both about "two" and "to define". Is the user required to set two values? Then why do I only see one listed as required in the table below?
2. There is a bullet list table, followed by a markdown formatted table and both include the same things. I assume the bullet list table should be removed as an outdated duplicate.

Reply:
You're right, I goofed and had left the bullets in there by mistake. Only the table is documented now.